internal/pkg/manifest/workload_ecs_test.go

```diff
@@ -271,15 +295,55 @@ func TestSecret_IsSecretsManagerName(t *testing.T) {
 	}
 }
 
+func TestSSMRequiresImport(t *testing.T) {
+	testCases := map[string]struct {
+		in     Secret
+		wanted bool
+	}{
+		"should return false if secret is plain": {
```

Comment:
Maybe one more test case for secret from `secretsmanager` name?

Reply:
added one more test case for secretsmanger